Bottom: 3a9ddd6c69294c3176dd0f7ab33e9de2fdbd669b
Top:    bba9aa08e0472542b58e875c909365d9df786c11
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 14:33:37 +0000

Refresh of move-quad-edge-stuff-into

---

diff --git a/src/hid/gtk/edge3d.c b/src/hid/common/edge3d.c
similarity index 100%
rename from src/hid/gtk/edge3d.c
rename to src/hid/common/edge3d.c
diff --git a/src/hid/gtk/edge3d.h b/src/hid/common/edge3d.h
similarity index 100%
rename from src/hid/gtk/edge3d.h
rename to src/hid/common/edge3d.h
diff --git a/src/hid/gtk/face3d.c b/src/hid/common/face3d.c
similarity index 100%
rename from src/hid/gtk/face3d.c
rename to src/hid/common/face3d.c
diff --git a/src/hid/gtk/face3d.h b/src/hid/common/face3d.h
similarity index 100%
rename from src/hid/gtk/face3d.h
rename to src/hid/common/face3d.h
diff --git a/src/hid/gtk/object3d.c b/src/hid/common/object3d.c
similarity index 100%
rename from src/hid/gtk/object3d.c
rename to src/hid/common/object3d.c
diff --git a/src/hid/gtk/object3d.h b/src/hid/common/object3d.h
similarity index 100%
rename from src/hid/gtk/object3d.h
rename to src/hid/common/object3d.h
diff --git a/src/hid/gtk/quad.c b/src/hid/common/quad.c
similarity index 100%
rename from src/hid/gtk/quad.c
rename to src/hid/common/quad.c
diff --git a/src/hid/gtk/quad.h b/src/hid/common/quad.h
similarity index 100%
rename from src/hid/gtk/quad.h
rename to src/hid/common/quad.h
diff --git a/src/hid/gtk/vertex3d.c b/src/hid/common/vertex3d.c
similarity index 100%
rename from src/hid/gtk/vertex3d.c
rename to src/hid/common/vertex3d.c
diff --git a/src/hid/gtk/vertex3d.h b/src/hid/common/vertex3d.h
similarity index 100%
rename from src/hid/gtk/vertex3d.h
rename to src/hid/common/vertex3d.h
